src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+h1 {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+  color: #000;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+}
+
+.card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.card__picture {
+  margin-top: 30px;
+}
+
+.card__info {
+  margin: 42px 15px 15px;
+}
+
+.card__title {
+  margin-bottom: 5px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__reviews {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  height: 15px;
+  width: 91px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__item {
+  content: url(./images/star.svg);
+}
+
+.stars__item:nth-child(-n+4) {
+  content: url(./images/star-active.svg);
+}
```

це тільки на 3 і 4 активних зірки, 
```suggestion
.stars--3 .stars__item:nth-child(-n+3),
.stars--4 .stars__item:nth-child(-n+4) {
  content: url(./images/star-active.svg);
}
```
а треба щоб були всі варіанти (на 1, на 2, на 3, на 4 і 5 активних зірок)

---

aaааааааа, дійшло тільки, зрозуміла